tap_trello/streams.py

```diff
@@ -240,18 +240,19 @@ def build_custom_fields_maps(self, **kwargs):
             custom_fields_map[custom_field['id']] = custom_field['name']
             if custom_field['type'] == 'list':
                 for dropdown_option in custom_field['options']:
-                    dropdown_options_map[dropdown_option['id']] = dropdown_option['value']['text']
+                    dropdown_options_map[dropdown_option['idCustomField'] + '_' + dropdown_option['id']] = dropdown_option['value']['text']
```

Comment:
This might be good to make a function like `_get_dropdown_option_key(self, field_id, option_id)` to keep the insert and get implementations symmetric.